src/models/pmv_ppd.js

```diff
@@ -0,0 +1,486 @@
+ * @param { number } vr - relative air speed, default in [m/s] in [fps] if `units` = 'IP'
+ * Note: vr is the relative air speed caused by body movement and not the air
+ * speed measured by the air speed sensor. The relative air speed is the sum of the
+ * average air speed measured by the sensor plus the activity-generated air speed
+ * (Vag). Where Vag is the activity-generated air speed caused by motion of
+ * individual body parts. vr can be calculated using the function `v_relative` which is in .utilities.js.
+ * @param { number } rh - relative humidity, [%]
+ * @param { number } met - metabolic rate
+ * @param { number } clo - clothing insulation
+ * Note: The activity as well as the air speed modify the insulation characteristics
+ * of the clothing and the adjacent air layer. Consequently, the ISO 7730 states that
+ * the clothing insulation shall be corrected {@link #ref_2|[2]}. The ASHRAE 55 Standard corrects
+ * for the effect of the body movement for met equal or higher than 1.2 met using
+ * the equation clo = Icl × (0.6 + 0.4/met) The dynamic clothing insulation, clo,
+ * can be calculated using the function `clo_dynamic` which is in .utilities.js.
+ * @param { number } [wme=0] - external work, default is 0
+ * @param { "ISO"|"ASHRAE" } [standard="ISO"] - comfort standard used for calculation
+ * - If "ISO", then the ISO Equation is used
+ * - If "ASHRAE", then the ASHRAE Equation is used
+ * Note: While the PMV equation is the same for both the ISO and ASHRAE standards,
+ * the ASHRAE Standard Use of the PMV model is limited to air speeds below 0.10m/s (20 fpm).
+ * When air speeds exceed 0.10 m/s (20 fpm), the comfort zone boundaries are adjusted based on the SET model.
+ * This change was introduced by the `Addendum C to Standard 55-2020`_
+ * @param { Pmv_ppdKwargs }kwargs - additional arguments
+ *
+ * @returns { Object } - Result of pmv and ppd
+ * @property { number } pmv - Predicted Mean Vote
+ * @property { number } ppd - Predicted Percentage of Dissatisfied occupants, [%]
+ *
+ * @example
+ * const tdb = 25;
+ * const tr = 25;
+ * const rh = 50;
+ * const v = 0.1;
+ * const met = 1.4;
+ * const clo = 0.5;
+ * // Calculate relative air speed
+ * const v_r = v_relative(v, met);
+ * // Calculate dynamic clothing
+ * const clo_d = clo_dynamic(clo, met);
+ * const results = pmv_ppd(tdb, tr, v_r, rh, met, clo_d);
+ * console.log(results); // Output: { pmv: 0.06, ppd: 5.1 }
+ * console.log(results.pmv); // Output: -0.06
+ */
+export function pmv_ppd(
+  tdb,
+  tr,
+  vr,
+  rh,
+  met,
+  clo,
+  wme = 0,
+  standard = "ISO",
+  kwargs = {},
+) {
+  const default_kwargs = {
+    units: "SI",
+    limit_inputs: true,
+    airspeed_control: true,
+  };
+  kwargs = Object.assign(default_kwargs, kwargs);
+
+  if (kwargs.units && kwargs.units === "IP") {
+    // Conversion from IP to SI units
+    const result = units_converter({ tdb, tr, vr }, "IP");
+    tdb = result.tdb;
+    tr = result.tr;
+    vr = result.vr;
```

Comment:
this is fine, but just so you know you can use the following syntax to do the same:
```javascript
({tdb, tr, vr} = units_converter({ tdb, tr, vr }, "IP"));
```

Reply:
done